androidApp/src/main/java/com/EENX15_22_17/digital_journal/android/ui/components/Checkbuttons.kt

```diff
@@ -66,7 +66,8 @@ fun <E : Enum<*>> EnumCheckboxesLazyGrid(
     onSelectionChanged: (selected: Set<E>) -> Unit,
     gridLayout: GridCells,
     isHorizontal: Boolean = false,
-    checkboxSpacing: Dp = 8.dp
+    checkboxSpacing: Dp = 8.dp,
+    isEnabled: Boolean = true
```

Comment:
I really would have thought this could be done via a modifier, but it seems that this is the only way :/